tests/NLog.UnitTests/LayoutRenderers/MessageTests.cs

```diff
@@ -219,5 +219,34 @@ public void MessageWithExceptionAndCustomSeparatorTest()
             logger.Debug(ex);
             AssertDebugLastMessage("debug", ex.ToString());
         }
+
+        [Fact]
+        public void SingleParameterException_OutputsSingleStackTrace()
+        {
+            // Stage
```

Comment:
Please use the term Arrange, -  AAA (Arrange-Act-Assert)

See https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80

Reply:
Fixed